neurom/apps/morph_stats.py

```diff
@@ -112,7 +116,14 @@ def extract_dataframe(neurons, config):
     if 'neuron' in config:
         del config['neuron']
 
-    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    with multiprocessing.Pool(n_workers) as pool:
```

Comment:
Can you make a `if/else` and only use Pool when `n_workers > 1`

Reply:
will do!